<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Derek Molloy DCU QT Application</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/photo.jpg.png</normaloff>../../Downloads/photo.jpg.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDial" name="lightLevelDial">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ledOnButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LED ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ledOffButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LED OFF</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pitchNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="rollSlider">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="pitchSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>21</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="pitchLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="rollLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="pitchLevelLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch Level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LEDRotLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LED On at Rotation Level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="accLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Accelerometer Data:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lightLevelEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ledAutoButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LED AUTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="rollLevelLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll Level:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="rollNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
